pom.xml

```diff
@@ -30,21 +30,21 @@
     <surefire.redirectTestOutputToFile>true</surefire.redirectTestOutputToFile>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <!-- version properties -->
-    <cdap.version>6.0.0-SNAPSHOT</cdap.version>
+    <cdap.version>6.0.0</cdap.version>
```

Comment:
shouldn't we rely on cdap 6.1.0-SNAPSHOT on develop similar to https://github.com/data-integrations/wrangler/blob/develop/pom.xml#L86?

Reply:
This plugin declares that it is compatible with 6.x versions of the data-pipeline app, which means this needs to be 6.0.0. If it were to use a new API in 6.1.0-SNAPSHOT and then were to be deployed on a 6.0.x version of CDAP, it would fail.

In general, I think we should change our process for plugins such that we don't update the dependent cdap version unless they need to use some new API, so that they're compatible with as many versions of CDAP as possible.